### PR TITLE
Add state flip task to Single-Qubit Gates kata

### PR DESCRIPTION
**Pull request by WWhitedogi**
Add state flip task to Single Qubit Gates

### CODE REVIEW

**Comment by WWhitedogi** [comment]
> @WWhitedogi please read the following Contributor License Agreement(CLA). If you agree with the CLA, please reply with the following information.
> 
> ```
> @microsoft-github-policy-service agree [company="{your company}"]
> ```
> 
> > Options:
> > 
> > * (default - no company specified) I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > 
> > * (when company given) I am making Submissions in the course of work for my employer (or my employer has intellectual property rights in my Submissions by contract or applicable law). I have permission from my employer to make Submissions and enter into this Agreement on behalf of my employer. By signing below, the defined term “You” includes me and my employer.
> > 
> > ```
> > @microsoft-github-policy-service agree company="Microsoft"
> > ```
> 
> Contributor License Agreement

@microsoft-github-policy-service agree

**Review by tcNickolas — CHANGES_REQUESTED** [review]
Looks great! I suggested a few small formatting improvements (you can just click "accept suggestions" on them for simplicity) and a file rename to make sure the build passes on all platforms.

Thank you!

File: katas/content/single_qubit_gates/state_flip/Solution.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+We can recognize that the Pauli $X$ gate will change the state $|0\rangle$ to $|1\rangle$ and vice versa, and $\alpha|0\rangle + \beta|1\rangle$ to $\alpha|1\rangle + \beta|0\rangle$.
```
[the remark this answers is not in the document]
Comment (WWhitedogi) [reply katas/content/single_qubit_gates/state_flip/Solution.md]: Got it! Already update.

**Review by tcNickolas — APPROVED** [review]
Looks great, thank you!

**Pull request merged**